app/models/inbox_scramble_set.rb

```diff
@@ -24,6 +24,18 @@ def backfill_round_information!
     self.round_type_id = matched_round.round_type_id
   end
 
+  def group_id
```

Comment:
The name here is a bit ambiguous. I don't think that the name `group_id` convincingly implies that it is non-numeric. What about something like `alphabetic_group_index`?